Tasks/VsTestV2/inputdatacontract.ts

```diff
@@ -50,6 +50,7 @@ export interface TfsSpecificSettings {
     ReleaseUri : string;
     ReleaseEnvironmentUri : string;
     WorkFolder : string;
+    PullRequestTargetBranchName : string;
```

Comment:
Since in the discussion today it came up that target branch name might be used beyond azure DevOps also it is better we create a separate section of optional inputs.

Preferably call it: TestSourceSettings under TestReportingSettings